test/unit/plugins/provisioners/chef/config/base_test.rb

```diff
@@ -82,4 +82,11 @@
       expect(subject.installer_download_path).to be(nil)
     end
   end
+
+  describe "#omnibus_url" do
+    it "defaults to https://omnitruck.chef.io" do
+      subject.finalize!
+      expect(subject.omnibus_url).to eq("https://omnitruck.chef.io")
+    end
```

Comment:
@briancain thanks! I added an additional test, so it looks now exactly like other chef options.

If you need/want more tests, could you please make a suggestion for implementation of this tests?
(I can't find any existing tests, which go completely through config options, capabilities, installer etc.)